python.rst

```diff
@@ -726,18 +731,14 @@ Exceptions
 
    move_to(x_coordinate, y_coordinate, speed_per_second)
 
-Classes
--------
-
-- Use Python "new-style" classes (must inherit from something, at
-  least ``object``) only. Do not use old-style classes.
 
 Strings
 -------
+
+- Prefer f-strings over ``.format()`` or ``%``.
```

Comment:
Might want to  explicitly caveat cases where the f-string has a complicated embedded function calls. This probably fits into minimizing line complexity in general, but probably want to avoid cases where multiple functions are called to transform a single variable.